spec/features/see_reviews_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'see reviews', %Q{
+  As a user,
+  I want to view the reviews for a beer
+  So that my mind can be influenced
+} do
+  # Acceptance Criteria
+  # * I must be on the details page for a specific beer
+  # * I must only see the reviews for the beer I’m viewing
+  # * I must see the reviews listed in order, most recent first
+
+  scenario 'sucessfully see reviews' do
+    beer_1 = FactoryGirl.create(:beer)
+    review_1 = Factorygirl.create(:review, beer: beer_1)
+    review_2 = Factorygirl.create(:review, beer: beer_1)
+    review_3 = Factorygirl.create(:review)
+
+    visit root_path
+    click_link beer_1.name
```

Comment:
After you make the above update, make sure the link points to the right beer with `review.beer`